tests/acceptance/certificates/delete-test.js

```diff
@@ -79,6 +79,8 @@ test(`visiting ${url} and deleting a certificate should delete the certificate`,
   signInAndVisit(url);
   andThen(function() {
     let deleteButton = find('button:contains(Delete)');
+    // Clobber window confirm to accept delete.
+    window.confirm = () => { return true; };
```

Comment:
This is my leeroy solution to keep the test runner happy... overwrite the native window method to return true without displaying the test-runner-blocking dialog.